pihole/datadog_checks/pihole/pihole.py

```diff
@@ -0,0 +1,106 @@
+from datadog_checks.base import AgentCheck, ConfigurationError
+
+
+class PiholeCheck(AgentCheck):
+    def __init__(self, name, init_config, instances):
+        super(PiholeCheck, self).__init__(name, init_config, instances)
+        for instance in instances:
+            host = instance.get('host')
+            if not host:  # Check if a host parameter exsists in conf.yaml
+                raise ConfigurationError('Error, please fix pihole.d/conf.yaml, host parameter is required')
```

Comment:
```suggestion
        host = self.instance.get('host')
        if not host:  # Check if a host parameter exsists in conf.yaml
            raise ConfigurationError('Error, please fix pihole.d/conf.yaml, host parameter is required')
```
That instances array always have only one in agent 6+

Reply:
👍